src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1140,6 +1142,63 @@ struct __StreamMetrics {
 
     // Last measured transfer rate in bytes per second
     UINT64 currentTransferRate;
+
+    // V1 metrics following
+
+    // Total stream duration - stream uptime
+    UINT64 uptime;
+
+    // Total transferred bytes
+    UINT64 transferredBytes;
+
+    // Total number of streaming sessions including new/errored/active
+    UINT64 totalSessions;
+
+    // Total number of active streaming sessions - only the ones that actually have streamed some data
+    UINT64 totalActiveSessions;

```

Comment:
do we need this? this should be at most 2.

Reply:
For devices perhaps. For NVR it could be hundreds

Reply:
Sorry, this is the sessions. This is an indicator of how many sessions had been active. Not the current ones.